lib/internal/abort_controller.js

```diff
@@ -357,6 +362,21 @@ function transferableAbortController() {
   return AbortController[kMakeTransferable]();
 }
 
+/**
+ * @param {AbortSignal} signal
+ * @param {any} resource
+ * @returns {Promise<void>}
+ */
+async function aborted(signal, resource) {
+  abortSignalValidator(signal, 'signal');
```

Comment:
```suggestion
  validateAbortSignal(signal, 'signal');
```

Reply:
Replacing this could cause a problem I think since a `validateAbortSignal` function already defined in the file but it throws `ERR_INVALID_THIS` rather than TypeError hence had imported validator using alias

Reply:
Should I cherry-pick your commit from that PR?

Reply:
Updating it here

Reply:
Done!